apps/web/components/getting-started/steps-views/ConnectedVideoStep.tsx

```diff
@@ -14,8 +18,13 @@ interface ConnectedAppStepProps {
   isPageLoading: boolean;
 }
 
-const ConnectedVideoStep = (props: ConnectedAppStepProps) => {
-  const { nextStep, isPageLoading } = props;
+const ConnectedVideoStepInner = ({
```

Comment:
This nested component allows for refactoring in the early return pattern, making it more readable and more predictable.